### DISCUSSION TITLE
Copilot doesn't work for python

### DISCUSSION THREAD

**Discussion by prkwa** [community/community#10214] [Copilot Conversations]
I just started programming, so I have a lot of inadequacies, but please help me.

Copilot doesn't work even though I have it enabled.
I've been able to use it for the last month or so without any problems, and I don't remember any changes to the specifications.
The version is up to date.

I'm not familiar with VSCODE itself, and I'd like to know where to start to check if I can get it to work.

**Comment by davecheney** [comment]
Can you please past the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Comment by Madheolii** [comment]
Berhenti dan keluar dari situs ini

Pada tanggal Sen, 17 Jan 2022 13.44, Dave Cheney ***@***.***>
menulis:
